packages/yoastseo/src/scoring/assessments/README.md

```diff
@@ -89,6 +89,9 @@ Below is a detailed overview of how scores for the readability assessments are c
 ### The overview of the Readability assessments scoring criteria
 Information on the scoring criteria for the available Readability assessments can be found [here](SCORING%20READABILITY.md).
 
+### The overview of the SEO assessments scoring criteria
+Information on the scoring criteria for the available SEO assessments can be found [here](https://github.com/Yoast/wordpress-seo/packages/yoastseo/src/scoring/assessments/SCORING%20SEO.md).
+
```

Comment:
```suggestion
```

Reply:
The link to the SEO assessment scoring criteria has been already added in this file (see line 66)